CHANGELOG.md

```diff
@@ -92,6 +92,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Always use `constant_score` query for `match_only_text` field ([#16964](https://github.com/opensearch-project/OpenSearch/pull/16964))
 - Fix Shallow copy snapshot failures on closed index ([#16868](https://github.com/opensearch-project/OpenSearch/pull/16868))
 - Fix multi-value sort for unsigned long ([#16732](https://github.com/opensearch-project/OpenSearch/pull/16732))
+- The `phone-search` analyzer no longer emits the international calling code as a token ([#16993](https://github.com/opensearch-project/OpenSearch/pull/16993))
```

Comment:
Seems like duplicate lines ..

Reply:
no, why? these are two different things. i just did them in the same PR since they're modifying the same sources.

Reply:
Oh I see, got it

Reply:
i've now merged it into once since i added a 3rd commit and think it's a bit pretentious to have three lines in the changelog for the same analyzer... 😅